lazy_theta_star_p_planner/include/lazy_theta_star_p_planner/lazy_theta_star_p.hpp

```diff
@@ -0,0 +1,237 @@
+// Copyright 2020 Anshumaan Singh
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LAZY_THETA_STAR_P_PLANNER__LAZY_THETA_STAR_P_HPP_
+#define LAZY_THETA_STAR_P_PLANNER__LAZY_THETA_STAR_P_HPP_
+
+#include <cmath>
+#include <vector>
+#include <queue>
+#include <algorithm>
+#include "rclcpp/rclcpp.hpp"
+#include "nav2_costmap_2d/costmap_2d_ros.hpp"
+
+typedef int id;
```

Comment:
I think we've talked about this before, but I think just calling these what they are in the code would make our lives alot easier. Just uding int / doubles would be really helpful for the average reader